giraffe/src/utils/PlotEnv.ts

```diff
@@ -148,6 +149,21 @@ export class PlotEnv {
     )
   }
 
+  public get yColumnType(): ColumnType {
+    for (let i = 0; i < this.config.layers.length; i++) {
+      const layer: any = this.config.layers[i]
```

Comment:
I know this works.. as long as we have a pin in to refactor this I'm ok with it.